### PR TITLE
:bug: Remove duplication of kairos vars in os-release

### LINKED ISSUE #1365: :bug: `/etc/os-release` with duplicated arguments when customizing disk image
[When customizing a disk image](https://kairos.io/docs/installation/raspberry/#customizing-the-disk-image) for the RPi, the `/etc/os-release` file gets appended with the new info of the OS but it never cleans the previous arguments

wrong:

```
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.17.3
PRETTY_NAME="Alpine Linux v3.17"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
KAIROS_NAME="kairos-core-alpine-arm-rpi"
KAIROS_VERSION="v2.0.2"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-core-alpine-arm-rpi"
KAIROS_VERSION_ID="v2.0.2"
KAIROS_PRETTY_NAME="kairos-core-alpine-arm-rpi v2.0.2"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
KAIROS_HOME_URL="https://github.com/kairos-io/kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/core-alpine-arm-rpi"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/kairos"
KAIROS_VARIANT="core"
KAIROS_FLAVOR="alpine-arm-rpi"KAIROS_NAME="kairos-alpine-arm-rpi"
KAIROS_VERSION="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-alpine-arm-rpi"
KAIROS_VERSION_ID="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_PRETTY_NAME="kairos-alpine-arm-rpi v2.0.2-k3sv1.26.4+k3s1"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues/new/choose"
KAIROS_HOME_URL="https://github.com/kairos-io/provider-kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/kairos-alpine-arm-rpi"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/provider-kairos"
KAIROS_VARIANT="core"
```

Expected:

```
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.17.3
PRETTY_NAME="Alpine Linux v3.17"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://gitlab.alpinelinux.org/alpine/aports/-/issues"
KAIROS_NAME="kairos-alpine-arm-rpi"
KAIROS_VERSION="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_ID="kairos"
KAIROS_ID_LIKE="kairos-alpine-arm-rpi"
KAIROS_VERSION_ID="v2.0.2-k3sv1.26.4+k3s1"
KAIROS_PRETTY_NAME="kairos-alpine-arm-rpi v2.0.2-k3sv1.26.4+k3s1"
KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues/new/choose"
KAIROS_HOME_URL="https://github.com/kairos-io/provider-kairos"
KAIROS_IMAGE_REPO="quay.io/kairos/kairos-alpine-arm-rpi"
KAIROS_IMAGE_LABEL="latest"
KAIROS_GITHUB_REPO="kairos-io/provider-kairos"
KAIROS_VARIANT="core"
```

### PR DESCRIPTION
**Pull request by mauromorales**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1365

### CODE REVIEW

**Review by Itxaka — DISMISSED** [review]
noice

**Review by Itxaka — APPROVED** [review]
brilliant! xD

**Comment by mauromorales** [comment]
I should have let you sign these commits LOL

**Comment by Itxaka** [comment]
> I should have let you sign these commits LOL

the I would get the blame when it breaks, no thanks :P

File: Earthfile [hunk 1]
```diff
@@ -80,6 +80,7 @@ OSRELEASE:
     ARG HOME_URL
 
     # update OS-release file
+    RUN sed -i -n '/KAIROS_/!p' /etc/os-release
```
Comment (jimmykarily) [inline Earthfile:83]: I guess this removed every line maching `KAIROS_`, right? It's worth a comment tbh, I don't find `sed` syntax to be very intuitive.

**Pull request merged**